hybrid/decomposers.py

```diff
@@ -849,35 +856,39 @@ def _kings_node_to_pegasus_chain(row: int, col: int) -> Tuple[
             return (0, x+1, 4, y), (1, y, 10, x)
 
 def _zephyr_to_chimeralike(coord: Tuple[int, int, int, int, int],
-                           t: int=4, half_offset: int=1) -> Tuple[
+                           t: int=4, half_offset: int=0) -> Tuple[
                                int, int, int, int]:
     """Map Zephyr[m,t] coordinates to Chimera[2m,2m,t]-style coordinates 
     
     Coordinates associated to a particular Chimera subgraph of 
     Zephyr are created.
-    Each Chimera cell is size 2t, t should be even
+    Each Chimera cell is size 2t, t should be even if half_offset=1
     Chimera-like coordinate
     (row, column, orientation (vertical/horizontal), in cell offset)
     is convenient for embedding enumeration and geometric displacements
     """
     u, w, k, j, z = coord
     row = u*w + (1-u)*(2*z + j)
     col = (1-u)*w + u*(2*z + j)
-    half_cell = t//2
-    return (row + half_offset*(u*(k//half_cell) + (1-u) - 1),
-            col + half_offset*((1-u)*(k//half_cell) + u - 1),
-            u, (k+half_offset*half_cell)%t)
+    if half_offset:
+        half_cell = t//2
+        return (row + half_offset*(u*(k//half_cell) + (1-u) - 1),
+                col + half_offset*((1-u)*(k//half_cell) + u - 1),
+                u, (k+half_offset*half_cell)%t)
+    else:
+        return (row, col, u, k)
```

Comment:
(minor) IMO cleaner to skip the else here.
```suggestion
    return (row, col, u, k)
```
Also seems to me like `half_offset` should be a boolean rather than an int, but that's not related to the changes in this PR, so probably out-of-scope.